src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -244,7 +243,7 @@ export const View = (props: Props) => {
           ),
         },
         elems: {
-          free_scan_link: <a href="/redesign/user/welcome" />,
+          start_free_scan_link: <a href="/redesign/user/welcome" />,
```

Comment:
Calling this out.. but I know this was implemented before this PR, but should it be `/welcome` or 

`redesign/user/dashboard/fix/data-broker-profiles/start-free-scan`